options/locale/locale_en-US.ini

```diff
@@ -2958,6 +2958,7 @@ config.mailer_sendmail_timeout = Sendmail Timeout
 config.mailer_use_dummy = Dummy
 config.test_email_placeholder = Email (e.g. test@example.com)
 config.send_test_mail = Send Testing Email
+config.send_test_mail_submit = Send
```

Comment:
Why not "global" translation `Send`?

Reply:
I don't see there are other places using the "Send"

And I am not sure whether the "Send" in "Send Mail" context could be translated in different languages for other "Send".

Reply:
Could label the translation `send_email` to make it more clear. But yes I agree this one does not seem like one that would have more use cases later. Still I think we should move more stuff to global, so it might be a good change nonetheless.

Reply:
If the key is "send_email", I would expect it's string is "Send E-mail", but not "Send"

Reply:
Hmm, then `email.send` but it would add another namespace, so also not ideal.

Reply:
Let's keep it then.